operator/helper/multiline.go

```diff
@@ -79,7 +79,9 @@ func NewLineStartSplitFunc(re *regexp.Regexp, flushAtEOF bool) bufio.SplitFunc {
 		if firstLoc == nil {
 			// Flush if no more data is expected
 			if len(data) != 0 && atEOF && flushAtEOF {
-				return len(data), data, nil
+				token = trimWhitespaces(data)
+				advance = len(token)
```

Comment:
`advance` is used to set the starting point for the next read. By advancing after trimming, I think we may be setting up the value such that we will reread the whitespace, which doesn't seem ideal. Unless I'm misunderstanding some part of this, then I would expect that we do:
```suggestion
				token = trimWhitespaces(data)
				advance = len(data)
```